backend/apps/posts/migrations/0004_alter_post_name.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.0.6 on 2022-07-20 21:09
+
+from django.db import migrations, models
```

Comment:
What have you changed the model schema? I find the migration file, but I can not find any changes in `backend/apps/posts/models.py`?